rancher/labels/index.md

```diff
@@ -24,6 +26,30 @@ Key | Value |Description
 
 > **Note:** For the labels prefixed with `io.rancher.scheduler.affinity`, there are slight variations based on your how want to match (i.e. equal or not equal, hard or soft rules). More details can be found [here]({{site.baseurl}}/rancher/rancher-compose/scheduling/#table-of-scheduling-labels).
 
+### Selector Format 
+
+The values for selector labels (i.e. `io.rancher.service.selector.link`, `io.rancher.service.selector.container`) can be in multiple formats to be able to select services/containers based on their labels. 
+
+```
+# Label could be key=value1 OR key=value2
+key in (value1, value2)
+# Label cannot be key=value1 OR key=value2
```

Comment:
"Label cannot be key=value1 OR key=value2" - here shouldn't be OR, but rather AND. So only services with label having key that is not a value1 AND not a value2, will be qualified.